onelogin/onelogin.saml.properties

```diff
@@ -0,0 +1,62 @@
+onelogin.saml2.strict = true
+onelogin.saml2.debug =  false
+
+###
+# Service Provider settings
+onelogin.saml2.sp.entityid = ${saml.sp.entity.id}
+onelogin.saml2.sp.assertion_consumer_service.url = ${saml.sp.acs.url}
+onelogin.saml2.sp.assertion_consumer_service.binding = urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST
+onelogin.saml2.sp.single_logout_service.url =
+onelogin.saml2.sp.single_logout_service.binding = urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect
+onelogin.saml2.sp.nameidformat = urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified
+onelogin.saml2.sp.x509cert =
+onelogin.saml2.sp.privatekey =
+
+###
+# Identity Provider settings
+onelogin.saml2.idp.entityid = ${saml.idp.entity.id}
+onelogin.saml2.idp.single_sign_on_service.url = ${saml.idp.sso.url}
+onelogin.saml2.idp.single_sign_on_service.binding = urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect
+onelogin.saml2.idp.single_logout_service.url =
+onelogin.saml2.idp.single_logout_service.response.url =
+onelogin.saml2.idp.single_logout_service.binding = urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect
+onelogin.saml2.idp.x509cert =
+# onelogin.saml2.idp.certfingerprint =
+# onelogin.saml2.idp.certfingerprint_algorithm = sha1
+
+###
+# Security settings
+onelogin.saml2.security.nameid_encrypted = false
+onelogin.saml2.security.authnrequest_signed = true
+onelogin.saml2.security.logoutrequest_signed = false
+onelogin.saml2.security.logoutresponse_signed = false
+onelogin.saml2.security.want_messages_signed =
+onelogin.saml2.security.want_assertions_signed = true
+onelogin.saml2.security.sign_metadata =
+onelogin.saml2.security.want_assertions_encrypted = ${saml.encryption.assertions.enabled}
+onelogin.saml2.security.want_nameid_encrypted = ${saml.encryption.nameid.enabled}
+onelogin.saml2.security.requested_authncontext = urn:oasis:names:tc:SAML:2.0:ac:classes:Password
+# todo should we enforce https password protection here?
+# (for https) onelogin.saml2.security.requested_authncontext = urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport
+onelogin.saml2.security.onelogin.saml2.security.requested_authncontextcomparison = exact
+onelogin.saml2.security.want_xml_validation = true
+
+# Algorithm that the toolkit will use on signing process. Options:
+#  'http://www.w3.org/2000/09/xmldsig#rsa-sha1'
+#  'http://www.w3.org/2000/09/xmldsig#dsa-sha1'
+#  'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256'
+#  'http://www.w3.org/2001/04/xmldsig-more#rsa-sha384'
+#  'http://www.w3.org/2001/04/xmldsig-more#rsa-sha512'
+onelogin.saml2.security.signature_algorithm = http://www.w3.org/2000/09/xmldsig#rsa-sha1
+#todo do we need to use rsa-sha256 here?
```

Comment:
Should we use a stronger signature algorithm here by default?
If we do, this will need to be tested with the Mock Idp & the real Portal